user-manual/transfer/import-metadata.rst

```diff
@@ -263,6 +264,199 @@ metadata. The JSON file is added to the metadata directory as above.
 
 The METS file will be populated exactly as in the CSV examples above.
 
+
+.. _metadata-xml-validation:
+
+Importing and validating descriptive metadata with source-metadata.csv
```

Comment:
I would suggest "Importing and validating xml metadata with source-metadata.csv" instead- the metadata could be non-descriptive in nature but it does have to be in xml format for this feature.